WcaOnRails/config/locales/de.yml

```diff
@@ -2227,6 +2236,10 @@ de:
       cannot_be_deleted_and_accepted: Eine Anmeldung kann nicht gleichzeitig akzeptiert und gelöscht werden.
       #original_hash: a38d024
       must_register: muss sich für mindestens eine Disziplin anmelden
+      #original_hash: abdc718
+      exceeds_event_limit:
+        one: Du darfst dich in genau einem Event anmelden
```

Comment:
Internationalize complained about an empty translation for "zero", however, the English baseline does not fill the "zero" entry, which make sense (the case where a competitor is allowed to register for _no events at all_ should not exist).
Therefore I went ahead in this particular case and first gave Internationalize a dummy placeholder and deleted the `zero:` line later a posteriori. Hope that's fine.